plugins/modules/vmware_vsan_release_catalog.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vmware_vsan_release_catalog
+version_added: '3.6.0'
```

Comment:
```suggestion
version_added: '3.5.0'
```

Due to the CI problems, I wasn't able to get this into 3.6.0. So it should be 3.7.0 here.